src/CFamily/CMake/Resources.resx

```diff
@@ -117,17 +117,26 @@
   <resheader name="writer">
     <value>System.Resources.ResXResourceWriter, System.Windows.Forms, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089</value>
   </resheader>
+  <data name="BadCMakeSettings" xml:space="preserve">
+    <value>[CompilationDatabaseLocator] Could not parse CMakeSettings.json: {0}</value>
```

Comment:
Are these message debug-only, or are some of them info?

We should agree on whether we are going to localise all debug messages or not.